src/app/modules/item/helpers/item-children.ts

```diff
@@ -0,0 +1,10 @@
+import { ItemChild } from '../http-services/get-item-children.service';
+
+export interface ItemChildWithAdditions extends ItemChild {
```

Comment:
I think this file should just be in the same directory as the `item-children-list` component... just to be used by component which uses it,  otherwise we'll end up with tons of mixed helpers.

Reply:
Fixed